modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/json/ProjectGraphJson.java

```diff
@@ -0,0 +1,25 @@
+package com.enonic.xp.admin.impl.rest.resource.project.json;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.google.common.collect.ImmutableList;
+
+import com.enonic.xp.project.ProjectGraph;
+
+public final class ProjectGraphJson
+{
+    private final List<ProjectGraphEntryJson> entries;
+
+    public ProjectGraphJson( final ProjectGraph projectGraph )
+    {
+        this.entries = projectGraph != null ? ImmutableList.copyOf( projectGraph.stream().
```

Comment:
Don't use guava when not necessarily, please. Unmodifiable list works here just fine